packages/access-client/src/agent.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-continue */
```

Comment:
is this needed here? I see the continue in `agent-data.js` and the global disabling of `no-continue` but don't see a continue in this file...

Reply:
good catch. I removed it now that the rule is not enabled more globally